### PR TITLE
Create basic_no_iqe.yaml

### PR DESCRIPTION
**Pull request by dscervantes**
Creating a basic  yaml without the iqe step. Just in case teams do not have an iqe plugin

### CODE REVIEW

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]
Can you to copy this PipelineRun: https://github.com/RedHatInsights/bonfire-tekton/blob/main/.tekton/simple-pipeline-test.yaml and create a new one that tests your new pipeline? Changing this [line](https://github.com/RedHatInsights/bonfire-tekton/blob/main/.tekton/simple-pipeline-test.yaml) to your new pipeline and removing these [lines](https://github.com/RedHatInsights/bonfire-tekton/blob/main/.tekton/simple-pipeline-test.yaml#L45-L48) and let's see if Konflux understands that it needs to run two checks

File: pipelines/basic_no_iqe.yaml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      default: rh-artifacts-bucket
+      description: Secret with connection details to S3
+    - name: BONFIRE_COMPONENT_NAME
+      type: string
+      default: ""
+      description: name of the app-sre component name
+    - name: COMPONENT_NAME
+      type: string
+      description: name of app-sre "resourceTemplate" in deploy.yaml for this component
+    - name: IQE_PLUGINS
```
Comment (ernesgonzalez33) [inline pipelines/basic_no_iqe.yaml:64]: @dscervantes I think you can delete several of these parameters if they are not being used in the tasks below

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: pipelines/basic_no_iqe.yaml (hunk 1 above)
Comment (mikedep333) [reply pipelines/basic_no_iqe.yaml:64]: My application's CI failed when using this PR
```
[User error] PipelineRun pulp-services-tenant parameters is missing some parameters required by Pipeline pulp-c8v4d-22nnb's parameters: pipelineRun missing parameters: [IQE_PLUGINS]
```

Comment (ernesgonzalez33) [reply pipelines/basic_no_iqe.yaml:64]: Yeah, that's because it doesn't have a default value here. But I think it can be even removed from the pipeline

**Review by ernesgonzalez33 — APPROVED** [review]

**Pull request merged**